TRayTracer/App.h

```diff
@@ -42,14 +43,17 @@ namespace rav
 		//Private constructor and destructors for Singleton
 		App();
 		~App();
-		
+
 		//GLFW Window
 		GLFWwindow* window;
 		VertexArray* screenQuadVAO;
 		VertexBuffer* screenQuadVBO;
 		GLuint defaultPrg;
+		PostProcess *postProcess;
```

Comment:
Name should self-explain that it is an abstract class to be decorated (aka. decorable)!

Reply:
This is not an abstract class :p 
but will change the name